doc/sphinx/azhelpgen/__init__.py

```diff
@@ -2,5 +2,3 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
-import pkg_resources
-pkg_resources.declare_namespace(__name__)
```

Comment:
Not sure why it declares namespace package. This does not look like a namespace package at all.